contracts/0.4.24/test_helpers/LidoPushableMock.sol

```diff
@@ -52,7 +52,7 @@ contract LidoPushableMock is Lido {
     }
 
     function initialize(address _oracle) public onlyInit {
-        _setOracle(_oracle);
+        _setDAOContracts(_oracle, _oracle, _oracle);
```

Comment:
Why not `_setDAOContracts(_oracle, address(0), address(0))`?

Reply:
There is a check that reverts tx if address is zero. Thus, let's keep it as is.